build-decision/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "build-decision"
+version = "1.0.0"
+description = "Administration of runtime configuration (Taskcluster settings) for Firefox CI"
+authors = [
+    { name = "Mozilla Release Engineering", email = "release+build-decision@mozilla.com" },
+]
+readme = "README.txt"
+requires-python = ">=3.11"
```

Comment:
This is ignored because the package is in a workspace. The only `requires-python` that matters then is the one at the root of said workspace.